UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -305,7 +310,20 @@
   LockBoxLib|MdeModulePkg/Library/LockBoxNullLib/LockBoxNullLib.inf
 !endif
   FileExplorerLib|MdeModulePkg/Library/FileExplorerLib/FileExplorerLib.inf
+
+!if $(SECURE_BOOT_ENABLE) == TRUE
+  RngLib|MdePkg/Library/BaseRngLibTimerLib/BaseRngLibTimerLib.inf

```

Comment:
BaseRngLib would be better, but it requires RDRAND support, so that'd probably break QEMU. https://github.com/9elements/edk2/commit/b3b1c5205a645377387452310ce355094c269363 is an option.